Makefile.am

```diff
@@ -61,6 +66,7 @@ if FATAL_WARNINGS
   COMMON_CXXFLAGS += -Werror
   COMMON_PROTOBUF_CXXFLAGS += -Werror -Wno-error=unused-parameter
```

Comment:
I think you need to add -Wno-error=deprecated-declarations here to fix the latest issue.